src/psyclone/tests/gocean1p0_config_test.py

```diff
@@ -363,23 +363,26 @@ def test_valid_config_files():
     invoke.schedule._const_loop_bounds = True
 
     gen = str(psy.gen)
-    new_loop1 = '''      DO j=1,2
-        DO i=3,4
+    new_loop1 = '''\
```

Comment:
Can the import at line 353 be moved to the top?

Reply:
Yes, done.